CMakeLists.txt

```diff
@@ -228,6 +228,11 @@ if(NOT SRPD_PLUGINS_PATH)
 endif()
 message(STATUS "SRPD plugins path:  ${SRPD_PLUGINS_PATH}")
 
+if(NOT SYSREPOCFG_DEFAULT_LYD_FORMAT)
+    set(SYSREPOCFG_DEFAULT_LYD_FORMAT LYD_XML)
+endif()
+message(STATUS "sysrepocfg default format: ${SYSREPOCFG_DEFAULT_LYD_FORMAT}")
```

Comment:
Not sure if this message is necessary, but fine, if you just add `sysrepocfg default data format: ...`.